lisa/testsuite.py

```diff
@@ -584,15 +566,20 @@ def start(
             "variables": copy.copy(case_variables),
         }
 
+        is_suite_continue = True
         #  replace to case's logger temporarily
-        suite_log = self.__log
-        (
-            is_suite_continue,
-            suite_error_message,
-            suite_error_stacktrace,
-        ) = self.__suite_method(
-            self.before_suite, test_kwargs=test_kwargs, log=suite_log
-        )
+        if hasattr(self, "before_suite"):
+            suite_log = self.__log
```

Comment:
the suite_log is used in other places, it should be initialized always.

Reply:
updated

Reply:
This line should be removed now.

Reply:
removed